django/contrib/postgres/search.py

```diff
@@ -1,3 +1,7 @@
+import re
+
+from psycopg2.extensions import adapt
```

Comment:
Have you looked into psycopg3 compatibility? We have some `if is_psycopg3:` branches.